components/raftstore/src/store/peer_storage.rs

```diff
@@ -1144,9 +1147,24 @@ pub mod tests {
             worker::{
                 FetchedLogs, LogFetchedNotifier, RaftlogFetchRunner, RegionRunner, RegionTask,
             },
+            Config,
         },
     };
 
+    const PENDING_APPLY_CHECK_INTERVAL: u64 = 200;
+    const STALE_PEER_CHECK_TICK: usize = 1;
+
+    fn make_raftstore_cfg(use_delete_range: bool) -> Arc<VersionTrack<Config>> {
```

Comment:
all these configuration except use_delete_range parameter are hardcoded. So what's the point of creating a config instead of hardcode in the RegionRunner ?

Reply:
Also please add a comment to clarify what's the consumer of this Config. It may be easily confused as the config used by raftstore itself.